workshop/Lab4/README.md

```diff
@@ -0,0 +1,74 @@
+# Lab 4 Deploy a service broker
+
+Learn to deploy a service broker, connect it to Kubernetes using Service Catalog,
+and view the catalog of services.
+
+# Open Service Broker API
+
+The [Open Service Broker API (OSBAPI)](https://www.openservicebrokerapi.org/) is an open
+standard that allows users of cloud platforms to provision, manage, and consume services
+in an automated fashion. We are going to create a service broker to automate deploying our
+Redis for us, as well as creating the secret that we consume in our deployment. If you're
+interested in what's going on behind the scenes, see [here](osbapi.md).
+
+# Deploy a broker
+
+For the purposes of this lab, we'll be using [Minibroker](https://github.com/osbkit/minibroker),
+a broker that lives inside our Kubernetes cluster. In the real world, the broker could be deployed
+within our cluster, or next to our cluster in the same data center, or out on the Internet, or anywhere
+in between. We'll be using Helm to install Minibroker, like so:
+
+```
+helm repo add minibroker https://minibroker.blob.core.windows.net/charts
+helm install --name minibroker --namespace minibroker minibroker/minibroker
+```
+
+You should be able to see the minibroker pods start up. Note that it's running in the `minibroker`
+namespace, which will also be the namespace it deploys subsequent resources into.
+```console
+$ kubectl get pods --all-namespaces
```

Comment:
maybe just have them do: `kubectl get pods -n minibroker` to keep the output easy